### DISCUSSION TITLE
How are CRD updated?

### DISCUSSION THREAD

**Discussion by shezaan** [bank-vaults/vault-operator#293] [Q&A] [answered]
As the question suggests, how do the crds get updated when deploying using helm? Since helm3 does not upgrade crds

**Comment by akijakya** [comment, marked as answer]
Hi @shezaan, you can upgrade the CRD to the version of your choice with `kubectl apply` before upgrading with Helm, or since `v1.21.0` you can also use kustomize, e.g. `kubectl kustomize https://github.com/bank-vaults/vault-operator/deploy/crd\?ref\=v1.21.0 | kubectl apply -f -`!

**Reply by Elyytscha** [reply]
this is sadly one of the main limitations resulting through utlizing helm and one of the reasons why olm > helm
